commit 31b1713e5d23407e16d5dddff30a73e3139902d4 (refs/changes/08/2208/4)
Author: Kody A Kantor <kody.kantor@gmail.com>
Date:   2017-09-07T21:18:54+00:00 (2 years, 1 month ago)
    
    MANTA-2955 audit jobs report false positives when mako dumps take over 16 hours
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: Joshua M. Clulow <jmc@joyent.com>

diff --git a/bin/upload_mako_ls.sh b/bin/upload_mako_ls.sh
index c3c5eee..658150f 100755
--- a/bin/upload_mako_ls.sh
+++ b/bin/upload_mako_ls.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright (c) 2014, Joyent, Inc.
+# Copyright (c) 2017, Joyent, Inc.
 #
 
 ###############################################################################
@@ -36,6 +36,7 @@ PID_FILE=/tmp/upload_mako_ls.pid
 TMP_DIR=/var/tmp/mako_dir
 LISTING_FILE=$TMP_DIR/$MANTA_STORAGE_ID
 MANTA_DIR=/mako
+START_TIME=`date -u +"%Y-%m-%dT%H:%M:%SZ"` # Time that this script started.
 
 
 
@@ -89,6 +90,7 @@ function manta_put() {
         -H "Date: $NOW" \
         -H "Authorization: Signature $AUTHZ_HEADER,signature=\"$SIGNATURE\"" \
         -H "Connection: close" \
+        -H "m-mako-dump-time: $START_TIME" \
         $MANTA_URL/$MANTA_USER/stor$1 \
         -T $2 \
         || fatal "unable to put $1"
